stdpopsim/cli.py

```diff
@@ -610,6 +746,13 @@ def write_simulation_summary(engine, model, contig, samples, seed=None):
     dry_run_text += f"{indent}Mean recombination rate: {mean_recomb_rate}\n"
     dry_run_text += f"{indent}Mean mutation rate: {mut_rate}\n"
     dry_run_text += f"{indent}Genetic map: {gmap}\n"
+    # Get the information about dfe if any
+    if dfe is not None:
+        dry_run_text += f"{indent}DFE: {dfe}\n"
+        if interval is not None:
```

Comment:
If we change the call to this function as I suggested above (passing in `intervals` not `args.intervals`) then we don't need this conditional here.

Reply:
I am following the rest of the way the code is written, should we change everything then?

Reply:
nevermind